tools/ReleaseTools.psm1

```diff
@@ -344,23 +401,29 @@ function New-DraftRelease {
         [ValidateSet([RepoNames])]
         [string]$RepositoryName,
 
-        [Parameter(ValueFromPipeline)]
+        [Parameter()]
         [string[]]$Assets
     )
     $Version = Get-Version -RepositoryName $RepositoryName
     $Changelog = (Get-FirstChangelog -RepositoryName $RepositoryName) -join "`n"
     $ReleaseParams = @{
-        Draft      = $true
         # NOTE: We rely on GitHub to create the tag at that branch.
-        Tag        = "v$Version"
-        Committish = "release/v$Version"
-        Name       = "v$Version"
-        Body       = $ChangeLog
-        PreRelease = [bool]$Version.PreReleaseLabel
-        OwnerName  = "PowerShell"
+        Tag            = "v$Version"
+        Committish     = "release/v$Version"
+        Name           = "v$Version"
+        Body           = $ChangeLog
+        Draft          = $true
+        PreRelease     = [bool]$Version.PreReleaseLabel
+        OwnerName      = "PowerShell"
         RepositoryName = $RepositoryName
+        WhatIf         = $WhatIfPreference
+        Confirm        = $ConfirmPreference
     }
 
     $Release = New-GitHubRelease @ReleaseParams
-    $Assets | New-GitHubReleaseAsset -Release $Release.Id
+    if ($Release) {
+        Write-Output "Draft release URL: $($Release.html_url)"
+        Write-Output "Uploading assets..."
```

Comment:
These should probably be either `Write-Host` or `Write-Verbose`

Reply:
Oh, sure.